c7n/resources/s3.py

```diff
@@ -667,9 +669,24 @@ def process_bucket(self, b):
 
 class BucketActionBase(BaseAction):
 
+    account_alias = None
+
     def get_permissions(self):
         return self.permissions
 
+    def get_std_format_args(self, bucket):
+        if not self.account_alias:
+            self.account_alias = get_account_alias_from_sts(
+                local_session(self.manager.session_factory))
+
+        return {
+            'account_id': self.manager.config.account_id,
+            'account': self.account_alias,
```

Comment:
can we skip the account alias here, besides the overhead of looking it up , its also a mutable value, so not per se the best for s3 output locations, also not everyone has it setup. aws standard for s3 outputs is account id.

Reply:
Yes, certainly for the `set-statements` action.